client/src/modules/user/AuthBox/components/PasswordChangeForm.jsx

```diff
@@ -0,0 +1,296 @@
+import React, { Fragment, PureComponent } from 'react';
+import { FormattedMessage, injectIntl } from 'react-intl';
+import PropTypes from 'prop-types';
+import _some from 'lodash/some';
+
+import AuthInput from './AuthInput';
```

Comment:
I suggest that relative imports should be at the bottom of the list as they are most likely the most related ones to the code below and the most specific for the particular modules.